
Dataset cannot be included due to GitHub push protection.

Use kagglehub to download:
import kagglehub
path = kagglehub.dataset_download("sid321axn/malicious-urls-dataset")
